sdccc/src/main/java/com/draeger/medical/sdccc/tests/glue/invariant/InvariantSubscriptionHandlingTest.java

```diff
@@ -0,0 +1,267 @@
+ * Copyright (c) 2023 Draegerwerk AG & Co. KGaA.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.draeger.medical.sdccc.tests.glue.invariant;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.draeger.medical.sdccc.configuration.EnabledTestConfig;
+import com.draeger.medical.sdccc.manipulation.precondition.impl.ConditionalPreconditions;
+import com.draeger.medical.sdccc.messages.MessageStorage;
+import com.draeger.medical.sdccc.messages.mapping.MessageContent;
+import com.draeger.medical.sdccc.sdcri.testclient.TestClient;
+import com.draeger.medical.sdccc.tests.InjectorTestBase;
+import com.draeger.medical.sdccc.tests.annotations.RequirePrecondition;
+import com.draeger.medical.sdccc.tests.annotations.TestDescription;
+import com.draeger.medical.sdccc.tests.annotations.TestIdentifier;
+import com.draeger.medical.sdccc.tests.util.ImpliedValueUtil;
+import com.draeger.medical.sdccc.tests.util.InitialImpliedValue;
+import com.draeger.medical.sdccc.tests.util.InitialImpliedValueException;
+import com.draeger.medical.sdccc.tests.util.MdibHistorian;
+import com.draeger.medical.sdccc.tests.util.NoTestData;
+import com.draeger.medical.sdccc.tests.util.guice.MdibHistorianFactory;
+import com.draeger.medical.sdccc.util.Constants;
+import com.draeger.medical.sdccc.util.TestRunObserver;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+import javax.xml.namespace.QName;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.somda.sdc.biceps.model.message.AbstractReport;
+import org.somda.sdc.biceps.model.message.DescriptionModificationReport;
+import org.somda.sdc.biceps.model.message.DescriptionModificationType;
+import org.somda.sdc.biceps.model.message.EpisodicAlertReport;
+import org.somda.sdc.biceps.model.message.EpisodicComponentReport;
+import org.somda.sdc.biceps.model.message.EpisodicContextReport;
+import org.somda.sdc.biceps.model.message.EpisodicMetricReport;
+import org.somda.sdc.biceps.model.message.EpisodicOperationalStateReport;
+import org.somda.sdc.biceps.model.message.ObservedValueStream;
+import org.somda.sdc.biceps.model.message.WaveformStream;
+import org.somda.sdc.biceps.model.participant.AbstractState;
+import org.somda.sdc.biceps.model.participant.DistributionSampleArrayMetricState;
+import org.somda.sdc.biceps.model.participant.RealTimeSampleArrayMetricState;
+import org.somda.sdc.biceps.model.participant.SampleArrayValue;
+import org.somda.sdc.dpws.soap.MarshallingService;
+import org.somda.sdc.dpws.soap.SoapMessage;
+import org.somda.sdc.dpws.soap.SoapUtil;
+import org.somda.sdc.dpws.soap.exception.MarshallingException;
+
+/**
+ * Glue Communication model binding tests.
```

Comment:
Both the Classname and the Requirement tell a different story...

Reply:
changed javadoc